README.md

```diff
@@ -1 +1,215 @@
-# go-module
\ No newline at end of file
+# Go-module
+
+This Go-module is intended as a template module for Fybrik written in [Go](https://go.dev/).
+
+This module features a read capability of data assets, using a generic server implementation written with [Gin Web Framework](https://pkg.go.dev/github.com/gin-gonic/gin) for Go.
+
+# How a Fybrik Application can access a dataset, using a Go module for Fybrik
+To see the Go module for Fybrik in action, you need to take these steps:
+1. Install Fybrik
+2. Register the Go-module to Fybrik
+3. Prepare a data asset and register it in a data catalog
+4. Define data access policy
+5. Deploy a Fybrik application
+6. Access the data asset using the Go-module 
+
+### Install fybrik
+Install Fybrik v1.3 using the [Quick Start](https://fybrik.io/v1.3/get-started/quickstart/), without the section of `Install modules`, and make sure to install Fybrik with Katalog as the data catalog.
+
+### Register the Go-module to Fybrik
+
+To register The Go-module as a Fybrik module apply `module.yaml` to the fybrik-system namespace of your cluster.
+
+To install the module, download the `module.yaml` from the repository and run:
```

Comment:
There is a command in the quick start that takes the module directly from the required repository (replace the arrow-flight repo with this one).